roles/nodeexporter/templates/node_exporter.service.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Node Exporter
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=prometheus
+ExecStart=/etc/node_exporter/node_exporter
```

Comment:
Should the value of ExecStart follow the `node_exporter_directory` variable?

Reply:
Added